pipeline/recon-pipeline.py

```diff
@@ -235,6 +235,16 @@ def do_scan(self, args):
         # luigi --module pipeline.recon.web.webanalyze WebanalyzeScan --target-file tesla --top-ports 1000 --interface eth0
         command = ["luigi", "--module", scans.get(args.scantype)[0]]
 
+        if args.target and args.target_file:
+            self.async_alert(style(f"[!] You can't use the --target and --target-file options in the same scan!"))
+            return
+        elif args.target:
+            with open("tmp_tgt", "w") as file:
```

Comment:
For trivially small writes like this, pathlib can be more succinct. 

`Path("my_tmp_file").write_text(args.target)`

It does the same things as the context manager.  You'll need a reference to your file for passing the path into the `arg_list`, so you can split that statement into two parts

```
tgt_file = Path("my_tmp_file")
tgt_file.write_text(args.target)
```